.github/workflows/test.yml

```diff
@@ -25,7 +25,16 @@ jobs:
       max-parallel: 4
       fail-fast: true
       matrix:
-        ruby: [2.4, 2.5, 2.6, 2.7]
+        ruby:
+          - '2.4'
+          - '2.5'
+          - '2.6'
+          - '2.7'
+        gemfile:
+          - gemfiles/rails_5.gemfile
+          - gemfiles/rails_6.gemfile
+    env:
+      BUNDLE_GEMFILE: ${{ matrix.gemfile }}
```

Comment:
Since I made a typo here, I fixed it and force pushed the changes prior to Pull Request reviews.